release notes/v0.38.0.md

```diff
@@ -0,0 +1,180 @@
+k6 v0.38.0 is here! 🎉
+
+## New Features!
+
+### Accessing the consolidated and derived options from the default function ([2493](https://github.com/grafana/k6/pull/2493))
+
+The `k6/execution` core module now lets you access the [consolidated and derived](https://k6.io/docs/using-k6/options/#where-to-set-options) options that k6 computed before it started executing the test. You can access consolidated options through the `exec.test.options` property. Note that consolidated options are frozen and cannot be modified.
+
+```javascript
+import exec from 'k6/execution';
+
+export const options = {
+  vus: 10,
+  duration : '30s'
+};
+
+export default function() {
+  console.log(exec.test.options.scenarios.default.vus) // 10
+}
+```
+
+### Dumping SSL keys to an NSS formatted key log file ([2487](https://github.com/grafana/k6/pull/2487))
+
+This release adds the ability to dump SSL keys while making TLS connections.
+You can then can use these keys to decrypt the whole traffic that k6 generates. 
+
+To do this, set the `SSLKEYLOGFILE` to some file path and run k6.
+This will populate the file with the keys.
+Then you can use Wireshark to capture the traffic, decrypt it, and use that for debugging. 
+
+[Here's an example that uses curl to inspect TLS traffic](https://daniel.haxx.se/blog/2018/01/15/inspect-curls-tls-traffic/).
+
+### AWS JSLib
+
+There's a new addition to the officially supported k6 JavaScript libraries: [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws).
+This library lets users interact with a selection of AWS services directly from their scrips. The library currently implements support for the [S3](https://k6.io/docs/javascript-api/jslib/aws/s3client/) and the [Secrets Manager](https://k6.io/docs/javascript-api/jslib/aws/secretsmanagerclient/) services.
+
+[The AWS JS lib documentaion](https://k6.io/docs/javascript-api/jslib/aws) has examples and details on how to use the library in your own scripts.
+###  Tagging by stages
+
+A few helper functions have been added to the [k6-jslib-utils](https://k6.io/docs/javascript-api/jslib/utils) library for tagging automatically all the metric's samples with the current running stage.
+## Breaking Changes
+
+### `console.log` and `console.info` pretty prints objects and arrays ([2375](https://github.com/grafana/k6/pull/2375))
+
+For convenience, all `console` methods such as `console.log` and `console.info` will now automatically `JSON.stringify` objects and arrays passed to it. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
```

Comment:
```suggestion
### `console` methods now pretty print objects and arrays ([2375](https://github.com/grafana/k6/pull/2375))

For convenience, all `console` methods such as `console.log()` and `console.info()` will now automatically `JSON.stringify()` objects and arrays passed to it. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
```